lib/bunq/resource.rb

```diff
@@ -6,6 +6,20 @@
 
 module Bunq
   class Resource
+    HEADER_ACCEPT = 'Accept'
```

Comment:
Nice improvement! We could also create a separate `Bunq::Header` module/class that contains all these constants.

Reply:
I inspired the idea in official Python SDK https://github.com/bunq/sdk_python/blob/master/bunq/sdk/http/api_client.py#L49. They store it in ApiClient. We do real http request making in Bunq::Resource, so I placed it here. Technically it is possible to migrate it to new module like this:

```ruby
module Bunq
  module Header
     ACCEPT = 'Accept'
 end
end

# Bunq::Header::Accept
```
I like the idea in general. @bforma Please approve!

Reply:
Sounds good.

Reply:
Done!

Reply:
Awesome!